### ISSUE TITLE
IMOS cheks for attributes being numeric fail incorrectly

### ISSUE THREAD

**Issue by mhidas** [aodn/compliance-checker#70]
E.g. for [this file](http://thredds.aodn.org.au/thredds/catalog/IMOS/eMII/checker_test/ABOS/Pulse/catalog.html?dataset=eMII/checker_test/ABOS/Pulse/IMOS_ABOS-SOTS_W_20150331T130000Z_Pulse_FV00_Pulse-11-MRU-Surface-wave-height-realtime_END-20150813T230000Z_C-20150814T021828ZZ.nc) we get:

```
    geospatial_lat_max                 :3:     0/ 1 :  
        check_attribute_type           :3:     0/ 1 : Attribute type is not
                                                      numeric
    geospatial_lat_min                 :3:     0/ 1 :  
        check_attribute_type           :3:     0/ 1 : Attribute type is not
                                                      numeric
```

but this is not true:

```
                :geospatial_lat_min = -46.94032f ;
                :geospatial_lat_max = -46.94032f ;
```

Looks like the code expects the type of these attributes to be `np.number`, which is incorrect (usually they will be `np.float32` or `np.float64`, which are subclasses of `np.number`)

**Comment by ggalibert** [comment]
Problem is that type could either be float32 or float64. Will see if I can combine both tests.

**Comment by mhidas** [comment]
@ggalibert Note that `check_attribute_type` allows `expected_type` to be given as a list.

**Comment by ggalibert** [comment]
@mhidas I created a PR https://github.com/aodn/compliance-checker/pull/72 to fix this issue.

**Comment by mhidas** [comment]
Fixed via #72
